diff -aurN libtheora-1.1.1.orig/lib/Makefile.am libtheora-1.1.1/lib/Makefile.am
--- libtheora-1.1.1.orig/lib/Makefile.am	2011-02-08 05:34:21.000000000 +0100
+++ libtheora-1.1.1/lib/Makefile.am	2011-02-08 05:45:43.000000000 +0100
@@ -153,6 +153,8 @@
 libtheoraenc_la_LDFLAGS = \
   -version-info @THENC_LIB_CURRENT@:@THENC_LIB_REVISION@:@THENC_LIB_AGE@ \
   @THEORAENC_LDFLAGS@ $(OGG_LIBS)
+libtheoraenc_la_LIBADD = \
+	libtheoradec.la
 
 libtheora_la_SOURCES = \
 	$(decoder_sources) \
@@ -163,7 +165,7 @@
   @THEORA_LDFLAGS@ @CAIRO_LIBS@ $(OGG_LIBS)
 
 debug:
-	$(MAKE) all CFLAGS="@DEBUG@" 
+	$(MAKE) all CFLAGS="@DEBUG@"
 
 profile:
 	$(MAKE) all CFLAGS="@PROFILE@"
